opensrp-reveal/build.gradle

```diff
@@ -215,7 +225,13 @@ dependencies {
     }
     implementation 'com.github.ybq:Android-SpinKit:1.2.0'
     implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-scalebar-v8:0.3.0'
+
+    // Repackaged jars will be placed here, adding them to classpath
+    //implementation fileTree(dir: "$project.buildDir/build/libs", include: ['*.jar'])
+    implementation fileTree(dir: "./build/libs", include: ['*.jar'])
```

Comment:
One question, what does this doe currently? I can't seem to get the folder created during the build

Reply:
let me remove this, it will be added on dynamic tasking PR where we are stripping out an unused class that uses `methodHandler.invoke` which is not supported below API 26 and not supported by desugaring. This was causing multidex to fail

Reply:
Cool, makes sense :+1: